README.md

```diff
@@ -4,7 +4,7 @@ See documentation [here](doc/00-overview.md)
 
 ``` yaml
 use-extension:
-  "@autorest/clicommon": "latest"
+  "@autorest/clicommon": "/Users/zhangqiaoqiao/work/code/autorest.cli.common"

```

Comment:
this shouldn't be here

Reply:
the latest clicommon that contains the directive feature Rodge is working on hasn't been released yet. Please ignore this for now.

Reply:
should have been fixed along with clicommon side